src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1204,9 +1207,16 @@ public void UpdateBubbleContent()
                 ? InfoBubbleViewModel.Style.Error
                 : InfoBubbleViewModel.Style.Warning;
 
+            style = NodeModel.State == ElementState.Info ? InfoBubbleViewModel.Style.Info : style;
+
             // NOTE!: If tooltip is not cached here, it will be cleared once the dispatcher is invoked below
             string content = NodeModel.ToolTipText;
-            if (string.IsNullOrWhiteSpace(content)) return;
+            if (string.IsNullOrWhiteSpace(content))
+            {
+                ErrorBubble.NodeMessages.Clear();
```

Comment:
When the tooltip message is empty or null, we need to clear the previous node messages, otherwise they will show up even the tooltip message is empty. Do you think this will change any previous behavior?